### ISSUE TITLE
Svelte: some `SvelteFlow` component events not strongly typed

### ISSUE THREAD

**Issue by cpsoinos** [xyflow/xyflow#4211]
### Describe the Bug

Some events on `SvelteFlow` are strongly typed, while others are not. For example, `"edgeclick"` is strongly typed, while `"nodedragstop"` is not:

`"edgeclick"` | `"nodedragstop"`
--- | ---
<img width="901" alt="image" src="https://github.com/xyflow/xyflow/assets/10081464/906ed9ef-fcc2-45ac-baf1-66a828239644"> | <img width="901" alt="image" src="https://github.com/xyflow/xyflow/assets/10081464/d7c2dcef-65b3-46d5-ba1e-559e6ea3ce26">

Digging down into the types shipped with the package in `node_modules/@xyflow/svelte/dist/lib/container/SvelteFlow/SvelteFlow.svelte.d.ts`, you can see why:
```ts
/// <reference types=".pnpm/svelte@4.2.1/node_modules/svelte" />
import { SvelteComponentTyped } from "svelte";
import type { SvelteFlowProps } from './types';
declare const __propDef: {
    props: SvelteFlowProps;
    events: {
        dragover: DragEvent;
        drop: DragEvent;
        paneclick: CustomEvent<{
            event: MouseEvent | TouchEvent;
        }>;
        panecontextmenu: CustomEvent<{
            event: MouseEvent;
        }>;
        edgeclick: CustomEvent<{
            edge: import("../..").Edge;
            event: MouseEvent | TouchEvent;
        }>;
        edgecontextmenu: CustomEvent<{
            edge: import("../..").Edge;
            event: MouseEvent;
        }>;
        nodeclick: CustomEvent<any>;
        nodemouseenter: CustomEvent<any>;
        nodemousemove: CustomEvent<any>;
        nodemouseleave: CustomEvent<any>;
        nodedragstart: CustomEvent<any>;
        nodedrag: CustomEvent<any>;
        nodedragstop: CustomEvent<any>;
        nodecontextmenu: CustomEvent<any>;
        selectionclick: CustomEvent<{
            nodes: import("../..").Node[];
            event: MouseEvent | TouchEvent;
        }>;
        selectioncontextmenu: CustomEvent<{
            nodes: import("../..").Node[];
            event: MouseEvent | TouchEvent;
        }>;
    } & {
        [evt: string]: CustomEvent<any>;
    };
    slots: {
        connectionLine: {
            slot: string;
        };
        default: {};
    };
};
type SvelteFlowProps_ = typeof __propDef.props;
export { SvelteFlowProps_ as SvelteFlowProps };
export type SvelteFlowEvents = typeof __propDef.events;
export type SvelteFlowSlots = typeof __propDef.slots;
export default class SvelteFlow extends SvelteComponentTyped<SvelteFlowProps_, SvelteFlowEvents, SvelteFlowSlots> {
}

```

### Steps to Reproduce the Bug or Issue

Attempt to get proper types for the `"nodedragstop"` event using `ComponentEvents<SvelteFlow>["nodedragstop"]`. You'll get `(property) nodedragstop: CustomEvent<any>`

### Expected behavior

As a developer, I expected to be able to get strongly typed `event`s for all events emitted by `SvelteFlow` by using Svelte's `ComponentEvents<SvelteFlow>` utility.

### Platform

- OS: [e.g. macOS, Windows, Linux]
- Browser: [e.g. Chrome, Safari, Firefox]
- Version: [e.g. 91.1]

**Comment by RedPhoenixQ** [comment]
Fixed in #4195

**Comment by moklick** [comment]
fixed in 0.1.0 - thanks @RedPhoenixQ
